functions.php

```diff
@@ -5,83 +5,60 @@
  * @package Go
  */
 
-/**
- * Theme constants.
- */
-define( 'GO_VERSION', '1.8.1' );
+// Load deactivation modal.
+define( 'GO_VERSION', '2.0.0' );
 define( 'GO_PLUGIN_DIR', get_template_directory( __FILE__ ) );
 define( 'GO_PLUGIN_URL', get_template_directory_uri( __FILE__ ) );
+require_once get_parent_theme_file_path( 'includes/class-go-theme-deactivation.php' );
 
 /**
- * AMPP setup, hooks, and filters.
+ * Sets up theme defaults and registers support for various WordPress features.
  */
-require_once get_parent_theme_file_path( 'includes/amp.php' );
+function go_theme_setup() {
+	add_theme_support( 'wp-block-styles' );
+	add_editor_style( 'style.css' );
 
-/**
- * Core setup, hooks, and filters.
- */
-require_once get_parent_theme_file_path( 'includes/core.php' );
+	// Disable CoBlocks Site Design controls.
+	delete_option( 'coblocks_site_design_controls_enabled' );
+}
```

Comment:
This is an interesting change here. This would handle a situation where users may have old CoBlocks and new Go for whatever reason.

Reply:
Yup! Never trust user inputs 😉